frontend/src/redux/reducers/index.ts

```diff
@@ -1,12 +1,14 @@
 import { combineReducers } from 'redux';
 import { userReducer } from './userReducer';
 import { todoReducer } from './todoReducer';
+import { userLoginReducer } from './userLoginReducer';
 import { popupLocationReduser } from './popupLocationReduser';
 import { locationsListReduser } from './locationListReduser';
 
 export const rootReducer = combineReducers({
   user: userReducer,
   todo: todoReducer,
+  userLogin: userLoginReducer,

```

Comment:
should be renamed

Reply:
eg. isUserAuthorized and isUserAuthorizedReducer